packages/tailwindcss-language-server/tests/fixtures/v4/workspaces/packages/style-export/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@private/style-export",
+  "exports": {
+    ".": {
+      "style": "./lib.css"
+    }
+  }
```

Comment:
Can we add a test that handles something like this:
```json
"exports": {
  "./theme": "./theme.css"
}
```

Reply:
Done